### DISCUSSION TITLE
Há alguma conferência em inglês no Brasil?

### DISCUSSION THREAD

**Discussion by Januszpl** [frontendbr/forum#1274] [Discussão]
Olá. Eu gostaria de saber se você tem alguma conferência internacional aqui no Brasil ou em conferências / reuniões locais se é possível ouvir e falar em inglês? Infelizmente o meu português neste momento não é muito bom para poder entender as conversas nessa língua.
(I've used google translator - but please do not answer in english. Let's keep it all portuguese ;))

**Comment by luizbills** [comment]
BrazilJS se me lembro bem tem várias (ou todas) palestras em inglês.

**Comment by victorferreira** [comment]
A [The Conf](https://www.theconf.club/) tem exatamente essa proposta.
